metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/utils/InfluxDbWriteObjectSerializer.java

```diff
@@ -84,8 +84,8 @@ private void formattedTime(Long time, TimeUnit precision, StringBuilder stringBu
     }
 
     private String escapeKey(String key) {
-        String toBeEscaped = SPACE.matcher(key).replaceAll("\\ ");
-        toBeEscaped = COMMA.matcher(toBeEscaped).replaceAll("\\,");
+        String toBeEscaped = SPACE.matcher(key).replaceAll("\\\\ ");
+        toBeEscaped = COMMA.matcher(toBeEscaped).replaceAll("\\\\,");
         return EQUAL.matcher(toBeEscaped).replaceAll("\\=");
```

Comment:
hmm, I'am looking at the docs https://docs.influxdata.com/influxdb/v0.13/write_protocols/line/#key and it seems that `=` is even not allowed in keys, only spaces and commas.